### DISCUSSION TITLE
Credits button is black for like 20 seconds before barotrauma logo shows up

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#13821] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

pressing "credits" on the main menu brings up the black screen, but no credits are visible for like 20 seconds, while waiting for the barotrauma logo to show up.

It may be better if the credits in the main menu did not autoscroll at all, so that they would be more easily read.

### Reproduction steps

1. press credits on the main menu
2. wait

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Couldn't repro, seems to take about 7 seconds which doesn't seem that excessive to me

https://github.com/FakeFishGames/Barotrauma/assets/14351070/8e694fc9-3571-4837-9ba2-f679e126e231

**Comment by Tsunder** [comment]
First name is only visible about 22 seconds after clicking "credits"
Logo showing up about 10 seconds after clicking, well, I don't know, it just seems like pointless extra waiting time to me.

https://github.com/FakeFishGames/Barotrauma/assets/5103517/4e8fdbc3-a533-4b51-9aec-799926d7c21d
